charts/argo-events/templates/eventbus-controller-deployment.yaml

```diff
@@ -74,3 +74,7 @@ spec:
       {{- if .Values.eventbusController.affinity }}
       affinity: {{ toYaml .Values.eventbusController.affinity | nindent 8 }}
       {{- end }}
+      {{- if .Values.eventbusController.resources }}
+      resources:
```

Comment:
`resources` is not a pod-level setting. Please move this inside the `containers` section.
And can you remove the conditional be more clean
```yaml
resources: {{- toYaml .Values.eventbusController.resources | nindent 12 }}
```